releasenotes-builder/src/test/java/com/github/checkstyle/templates/TemplateProcessorTest.java

```diff
@@ -252,6 +252,24 @@ public void testGenerateCustomTemplate() throws Exception {
         assertFile("customTemplate.txt", MainProcess.GITHUB_FILENAME);
     }
 
+    @Test
+    public void testVersionValidationForBreakingCompatibility() throws Exception {
+        final List<String> errors = MainProcess.runPostGenerationAndPublication(
+            createNotes(createAllNotes(), null, null, null, null), createBaseCliOptions()
+                .setValidateVersion(true).build());
+
+        Assert.assertEquals("no version validation error", 0, errors.size());
```

Comment:
ok, now add tests methods that will have some errors.

Reply:
By doing:   `.setValidateVersion(false).build());` and   `Assert.assertEquals("no version validation error", 1, errors.size());` ?

Reply:
yes, see how to set special flags and how to make release notes of different form:
https://github.com/checkstyle/contribution/blob/1b3e33600898a849791dc88c90df2bb3f80400e7/releasenotes-builder/src/test/java/com/github/checkstyle/templates/TemplateProcessorTest.java#L233-L246

Reply:
Changes Done